### ISSUE TITLE
Magnetic calculation not working

### ISSUE THREAD

**Issue by mystery-e204** [phonopy/phelel#25]
I'm on
- spglib 2.5.0
- phelel 0.6.1 (current develop, SHA: 5cda44abe74c919a1ffdd60375516013a1cb737e)
- phonopy 2.28.0-dev14+gd3bb990d

I try to run phelel through velph in a magnetic system. I created the velph.toml by specifying `--magmom`. The velph.toml and phelel_disp.yaml files also contain the correct on-site magnetic moments. After the supercell calculation, I try to run
```bash
velph phelel differentiate
```
but I run in to the following error
```bash
Traceback (most recent call last):
  File "/path/micromamba/envs/phelel/bin/velph", line 8, in <module>
    sys.exit(cmd_root())
             ^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/micromamba/envs/phelel/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/src/phelel/src/phelel/velph/cli/phelel/__init__.py", line 171, in cmd_differentiate
    phe.save_hdf5(filename=hdf5_filename)
  File "/path/src/phelel/src/phelel/api_phelel.py", line 445, in save_hdf5
    write_phelel_params_hdf5(
  File "/path/src/phelel/src/phelel/file_IO.py", line 42, in write_phelel_params_hdf5
    _add_datasets(
  File "/path/src/phelel/src/phelel/file_IO.py", line 367, in _add_datasets
    "spacegroup_number", data=int(symmetry_dataset.number), dtype="int_"
                                  ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'SpglibMagneticDataset' object has no attribute 'number'
```

**Comment by atztogo** [comment]
Thanks @mystery-e204 for your report. I think it is fixed by PR #26 which is merged to develop branch.

**Comment by mystery-e204** [comment]
Thanks! I can confirm that it's working now.
